libvast/include/vast/arrow_table_slice.hpp

```diff
@@ -71,8 +71,10 @@ class arrow_table_slice final {
   /// @param slice The encoding-specific FlatBuffers table.
   /// @param parent The surrounding chunk.
   /// @param batch A pre-existing record batch.
+  /// @param schema VAST schema matching the record batch schema.
```

Comment:
Is the idea that the new parameter coerces the provided batch into a desired schema?

Reply:
No, this is purely an optimization - we often have many record batches from the same schema (single feather / parquet / segment store), and don't want to recompute the VAST schema in the constructor every time.